### ISSUE TITLE
Password reset error

### ISSUE THREAD

**Issue by Derkades** [NamelessMC/Nameless#1963]
```
[2020-12-13, 12:54:39] /home/examplel/public_html/core/classes/DB.php(76): Uncaught PDOException: SQLSTATE[42000]: Syntax error or access violation: 1064 You have an error in your SQL syntax; check the manual that corresponds to your MariaDB server version for the right syntax to use near '' at line 1 in /home/examplel/public_html/core/classes/DB.php:76
Stack trace:
#0 /home/examplel/public_html/core/classes/DB.php(76): PDOStatement->execute()
#1 /home/examplel/public_html/core/classes/DB.php(196): DB->createQuery('UPDATE nl2_user...', Array)
#2 /home/examplel/public_html/core/classes/User.php(79): DB->update('nl2_users', NULL, Array)
#3 /home/examplel/public_html/modules/Core/pages/forgot_password.php(107): User->update(Array)
#4 /home/examplel/public_html/index.php(84): require('/home/examplel/...')
#5 {main}
```

Is it a problem that the `$id` parameter to the `update` function is not specified (and defaults to null)?

**Comment by Derkades** [comment]
Fixed in ab71468 but it seems to be like there is a better way

**Comment by samerton** [comment]
Fix seems suitable since the new user system was implemented
